db/seeds.rb

```diff
@@ -6,31 +6,68 @@
-Language.find_or_create_by({id:1,name:'English',code:'EN'})
-Language.find_or_create_by({id:2,name:'Español',code:'ES'})
-Language.find_or_create_by({id:3,name:'Français',code:'FR'})
+
+Language.find_or_create_by(id: 1, name: 'English', code: 'EN')
+Language.find_or_create_by(id: 2, name: 'Español', code: 'ES')
+Language.find_or_create_by(id: 3, name: 'Français', code: 'FR')
 Language.connection.execute("select setval('languages_id_seq', (select max(id) from languages))")
 
 
-# note, these hardcoded id's are needed to match the migrated data
-Currency.find_or_create_by({id:1,name:'Argentinean Peso',code:'ARS',symbol:'AR$'})
-Currency.find_or_create_by({id:2,name:'U.S. Dollar',code:'USD',symbol:'US$'})
-Currency.find_or_create_by({id:3,name:'British Pound',code:'GBP',symbol:'GB£'})
-Currency.find_or_create_by({id:4,name:'Nicaraguan Cordoba',code:'NIO',symbol:'NI$'})
+Currency.find_or_create_by(id: 1, name: 'Argentinean Peso', code: 'ARS', symbol: 'AR$')
+Currency.find_or_create_by(id: 2, name: 'U.S. Dollar', code: 'USD', symbol: 'US$')
+Currency.find_or_create_by(id: 3, name: 'British Pound', code: 'GBP', symbol: 'GB£')
+Currency.find_or_create_by(id: 4, name: 'Nicaraguan Cordoba', code: 'NIO', symbol: 'NI$')
 Currency.connection.execute("select setval('currencies_id_seq', (select max(id) from currencies))")
 
 
-# note, these hardcoded id's are needed to match the migrated data
-Country.find_or_create_by({id:1,name:'Argentina',iso_code:'AR',default_language_id:2,default_currency_id:1})
-Country.find_or_create_by({id:2,name:'Nicaragua',iso_code:'NI',default_language_id:2,default_currency_id:4})
-Country.find_or_create_by({id:3,name:'United States',iso_code:'US',default_language_id:1,default_currency_id:2})
+Country.find_or_create_by(id: 1, name: 'Argentina', iso_code: 'AR', default_language_id: 2, default_currency_id: 1)
+Country.find_or_create_by(id: 2, name: 'Nicaragua', iso_code: 'NI', default_language_id: 2, default_currency_id: 4)
+Country.find_or_create_by(id: 3, name: 'United States', iso_code: 'US', default_language_id: 1, default_currency_id: 2)
 Country.connection.execute("select setval('countries_id_seq', (select max(id) from countries))")
 
+
 # for now mapping the '0' Person refs to 'null' and allowing null refs in the schema
 # Person.find_or_create_by({id:0, name: 'dummy', first_name: 'dummy', division_id: Division.root_id})
+
+# handy temporary change for debugging
+# Option.destroy_all
+# OptionSet.destroy_all
+
+loan_status = OptionSet.find_or_create_by(division: Division.root, model_type: Loan.name, model_attribute: 'status')
+# beware rerunning these different position values, will result in duplicated data
+# a more verbose approach would be needed if that is important to support
+# also we can consider dropping the 'position' assignments until an ordering different than 'value' is needed
+loan_status.options.find_or_create_by(value: 1, position: 1).set_label_list(en: 'Active', es: 'Prestamo Activo')
```

Comment:
Mel is going to think on this structure a bit more and see if she can come up with an improvement. My inclination at this point is that we should try to make the code agnostic of any particular options, including "active" and "completed", and to the extent that they need to be treated as special, they should be marked as special in the database. But I'm holding off on recommending that change until I hear back from Mel. 

I am concerned about this `value` column, though. I don't understand why it's there, or why you are specifying its values. I prefer not to set IDs explicitly either, but since you're doing that above anyway I don't know why you're treating these differently. If `value` is just an integer, why not use ID and leverage rails associations? If it's meaningful, why not make it meaningful, like with slugs/internal names? Again, I lean toward the former.